Gems/Atom/Feature/Common/Assets/Passes/SkyAtmosphere.pass

```diff
@@ -49,6 +49,20 @@
                         }
                     ]
                 },
+                {
+                    "Name": "SkyVolumeLUTPass",
```

Comment:
While the LUT pass is conceptually part of the "Sky atmosphere" system, it may make sense to overlap this in async compute much earlier in the frame (for likely more impressive runtime savings). The pass looks to be almost entirely ALU bound (check) so this would be a good candidate to overlap the depth prepass or motion vector passes, give that those are typically rasterizer-bound. Overlap with other work that is bound by the memory units could work also (generally, ALU-bound work is the easiest to shuffle around since most work is not ALU bound sadly).

Reply:
This can be added as a GHI to be added later on. RHI does support async compute but requires gpu captures to ensure it is working correctly and getting scheduled at the correct time.